presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -242,7 +243,7 @@ protected AggregationNode build()
 
     public ApplyNode apply(Assignments subqueryAssignments, List<Symbol> correlation, PlanNode input, PlanNode subquery)
     {
-        return new ApplyNode(idAllocator.getNextId(), input, subquery, subqueryAssignments, correlation);
+        return new ApplyNode(idAllocator.getNextId(), input, subquery, subqueryAssignments, correlation, new NullLiteral());
```

Comment:
Making this `Optional` because of tests, does not sound to me as a good idea. I will use comment.